src/elements/form/Control.ts

```diff
@@ -99,7 +99,11 @@ import { Helpers } from './../../utils/Helpers';
             <div class="error-message">
                 <span *ngIf="isDirty && errors?.required">{{control.label | uppercase}} is required</span>
                 <span *ngIf="isDirty && errors?.minlength">{{control.label | uppercase}} is required to be a minimum of {{ control.minlength }} characters</span>
-                <span *ngIf="isDirty && errors?.maxlength">{{control.label | uppercase}} is required to be a maximum of {{ control.maxlength }} characters</span>
+                <span class="character-limit-error" *ngIf="isDirty && maxLengthMet && focused && !errors?.maxlength">Sorry, you have reached the maximum character count of {{ control.maxlength }} for this field</span>
+                <span class="character-limit-error" *ngIf="errors?.maxlength">Sorry, you have exceeded the maximum character count of {{ control.maxlength }} for this field</span>
+                <span class="text-field-character-limit" [attr.data-automation-id]="text-area-character-count" *ngIf="focused && control.maxlength && (control.controlType=='text-area' || control.controlType=='textbox')">
```

Comment:
This should also always appear to the right even without an error.

![screenshot 2017-01-20 10 09 51](https://cloud.githubusercontent.com/assets/870799/22156396/abf38002-def8-11e6-9b34-39e0eb16ed14.png)
![screenshot 2017-01-20 10 09 58](https://cloud.githubusercontent.com/assets/870799/22156401/b25a3bc0-def8-11e6-8084-208a12eaf997.png)
![screenshot 2017-01-20 10 10 05](https://cloud.githubusercontent.com/assets/870799/22156399/b0a1cbea-def8-11e6-8577-62e56f31d53d.png)

it should always be on the bottom right even with error and without error